plugins/woocommerce/client/admin/client/settings-payments/components/incentive-banner/incentive-banner.scss

```diff
@@ -0,0 +1,32 @@
+.woocommerce-incentive-banner {
+	background-color: $studio-woocommerce-purple-0;
+	margin: 32px;	// TODO: correctly import the $content-spacing-large variable
+	box-shadow: none;
+	border-radius: 8px;
+
+	&__content {
+		display: flex;
+		align-items: center;
+	}
+
+	&__body {
+		flex: 1;
+
+		& > h2 {
+			font-size: 24px;
+			line-height: 32px;
+			color: $gray-900; // #333333
```

Comment:
I think we don't need this comments since variable values may change

Reply:
Agreed. Removed!